next-cloudinary/src/components/CldOgImage/CldOgImage.tsx

```diff
@@ -51,24 +64,24 @@ const CldOgImage = ({ excludeTags = [], twitterTitle, ...props }: CldOgImageProp
 
     return (
     <Head>
-      <meta property="og:image" content={ogImageUrl} />
-      <meta property="og:image:secure_url" content={ogImageUrl} />
-      <meta property="og:image:width" content={`${options.width}`} />
-      <meta property="og:image:height" content={`${options.height}`} />
+      <meta key={metaKeys['og:image']} property="og:image" content={ogImageUrl} />
+      <meta key={metaKeys['og:image:secure_url']} property="og:image:secure_url" content={ogImageUrl} />
+      <meta key={metaKeys['og:image:width']} property="og:image:width" content={`${options.width}`} />
+      <meta key={metaKeys['og:image:height']} property="og:image:height" content={`${options.height}`} />
 
       {options.alt && (
-        <meta property="og:image:alt" content={options.alt} />
+        <meta key={metaKeys['og:image:alt']} property="og:image:alt" content={options.alt} />
       )}
 
       {/* Required for summary_large_image, exclude vai excludeTags */}
       {/* https://developer.twitter.com/en/docs/twitter-for-websites/cards/overview/summary-card-with-large-image */}
 
       {!excludeTags.includes('twitter:title') && (
```

Comment:
Is there a reason this is the only property that is allowed to be excluded?

Reply:
That should have been line 66 I think...referring to 'twitter:title'

Reply:
only because i created a mechanism for it rather than a 1-off that hasnt had additional purpose yet lol

the issue is that you MUST have a twitter title so if one doesnt exist i add basically an empty one, but if you wanna manage it yourself, i let you exclude it

open to ideas, maybe in a diff ticket though